be1-go/channel/election/mod.go

```diff
@@ -228,31 +228,21 @@ func (c *Channel) Broadcast(broadcast method.Broadcast, socket socket.Socket) er
 }
 
 // handleMessage handles a message received in a broadcast or publish method
-func (c *Channel) handleMessage(msg message.Message, socket socket.Socket) error {
-
-	jsonData, err := base64.URLEncoding.DecodeString(msg.Data)
-	if err != nil {
-		return xerrors.Errorf(failedToDecodeData, err)
-	}
+func (c *Channel) handleMessage(msg message.Message) error {
 
-	object, action, err := messagedata.GetObjectAndAction(jsonData)
+	err := c.registry.Process(msg)
 	if err != nil {
-		return xerrors.Errorf("failed to get object and action: %v", err)
+		return xerrors.Errorf("failed to process message: %w", err)
```

Comment:
```suggestion
		return xerrors.Errorf("failed to process message: %v", err)
```

No need to wrap the error